docs/ota-client-guide/modules/ROOT/pages/posix-secondaries-bitbaking.adoc

```diff
@@ -14,7 +14,7 @@ It is assumed that a reader is familiar with Yocto and bitbaking in general as w
 details of which are out of scope of this doc and can be found in the following guides:
 
 * link:https://github.com/advancedtelematic/meta-updater/blob/master/README.adoc[meta-updater README].
-* https://docs.ota.here.com/quickstarts/qemuvirtualbox.html[How to build 'core-image-minimal' image for QEMU]
+* https://docs.ota.here.com/ota-client/dev/build-quemu.html[How to build 'core-image-minimal' image for QEMU]
```

Comment:
Should be an xref

Reply:
Had to add some conditional logic, since xref points to an almost empty .adoc file - it just contains an include (which would be useless if viewed on Github). If viewed in Github, still HTML link is used.